drools-wb-services/drools-wb-verifier/drools-wb-verifier-web-worker/src/main/java/org/drools/workbench/verifier/webworker/client/Poster.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.drools.workbench.verifier.webworker.client;
+
+import org.drools.workbench.services.verifier.api.client.Status;
+import org.drools.workbench.services.verifier.api.client.reporting.Issues;
+import org.drools.workbench.services.verifier.plugin.client.Logger;
+import org.drools.workbench.services.verifier.plugin.client.api.WebWorkerException;
+import org.drools.workbench.services.verifier.plugin.client.api.WebWorkerLogMessage;
+import org.jboss.errai.enterprise.client.jaxrs.MarshallingWrapper;
+
+public class Poster {
```

Comment:
LOL.. finally after scanning millions of lines of code; I find what I'm looking for :-)